src/NHibernate.Test/TypesTest/GenericTypeFixtureBase.cs

```diff
@@ -0,0 +1,280 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using NHibernate.Cfg;
+using NHibernate.Linq;
+using NHibernate.Linq.Visitors;
+using NHibernate.Mapping.ByCode;
+using NHibernate.Type;
+using NUnit.Framework;
+
+namespace NHibernate.Test.TypesTest
+{
+	/// <summary>
+	/// Base class for fixtures testing individual types, created to avoid
+	/// code duplication in derived classes.
+	/// </summary>
+	public abstract class GenericTypeFixtureBase<TProperty, TType> : TestCase where TType : IType
+	{
+
+		protected override string MappingsAssembly
```

Comment:
nit: undue blank line.


```suggestion
	{
		protected override string MappingsAssembly
```